src/components/ArticlePage.jsx

```diff
@@ -10,16 +12,34 @@ import Comments from './Comments';
 function ArticlePage() {
     const {article_id} = useParams();
     const [article, setArticle] = useState();
+    const [voteCount, setVoteCount] = useState(0);
     const [loading, setLoading] = useState(true);
+    const [vote, setVote] = useState(false);
+    const [downVote, setDownVote] = useState(false);
 
     useEffect(() => {
         fetchArticleById(article_id)
         .then((articleResponse) => {
             setArticle(articleResponse);
+            setVoteCount(articleResponse.votes);
             setLoading(false)
         })
     }, [])
 
+    function incrementVote(){
+        setVote(true);
+        setDownVote(false);
+        setVoteCount(1);
+        patchVotes(article_id, 1);
```

Comment:
Make sure you are setting your vote amount before you make your patch request. The vote amount itself should be stored in state and updated each time your incrementVote or decrementVote function is invoked. It looks like you are currently changing the vote amount directly in your render on line 58 which may explain some of the strange behaviour of your code.